src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -37,6 +81,9 @@ public void CheckProjectReferenceAlias ()
 				using (var b = CreateApkBuilder (Path.Combine (path, proj.ProjectName), cleanupAfterSuccessfulBuild: false, cleanupOnDispose: false)) {
 					b.ThrowOnBuildFailure = false;
 					Assert.IsTrue (b.Build (proj), "Project should have built.");
+					string text = File.ReadAllText (Path.Combine (Root, path, proj.ProjectName, "Resources", "Resource.designer.cs"));
+					Assert.IsTrue (text.Contains ("extern alias Lib1"), "Resource.designer.cs should contain  extern alias Lib1.");
+					Assert.IsFalse (text.Contains ("extern alias Lib2"), "Resource.designer.cs should contain  extern alias Lib2.");
```

Comment:
I think the `Assert.IsFalse()` here might not be testing anything too interesting? I think the the test would need a `Build()` call for `library2.ProjectName` to create a project directory and files for `library2` so that the project reference from the application project would have files available to reference. Then the assert could be an `Assert.IsTrue()`.

Maybe it would be OK to clean this up by removing this test and adding the corresponding asserts into `CheckMultipleLibraryProjectReferenceAlias` instead?

Sketch of the change to `CheckMultipleLibraryProjectReferenceAlias()`:

```diff
Assert.IsTrue (b.Build (proj), "Project should have built.");
string[] text = File.ReadAllLines (Path.Combine (Root, path, proj.ProjectName, "Resources", "Resource.designer.cs"));
Assert.IsTrue (text.Count (x => x.Contains ("Library1.Resource.String.library_name")) == 2, "library_name resource should be present exactly once for each library");
-Assert.IsTrue (text.Count (x => x == "extern alias Lib1A;" || x == "extern alias Lib1B;") <= 1, "No more than one extern alias should be present for each library.");
+Assert.IsTrue (text.Contains ("extern alias Lib1A;"), "Resource.designer.cs should contain extern alias Lib1A.");
+Assert.IsFalse (text.Contains ("extern alias Lib1B;"), "No more than one extern alias should be present for each library.");
+Assert.IsTrue (text.Contains ("extern alias Lib2A;"), "Resource.designer.cs should contain extern alias Lib2A.");
+Assert.IsFalse (text.Contains ("extern alias Lib2B;"), "No more than one extern alias should be present for each library.");
```

Reply:
done :)